tests/ArmTemplates.Tests/Extractor/Abstractions/ExtractorMockerTestsBase.cs

```diff
@@ -74,6 +77,69 @@ public abstract class ExtractorMockerTestsBase : TestsBase
                 ParamBackend = MockParameterizeBackendSettings.ToString(),
                 ExtractGateways = MockExtractGateways.ToString()
             };
-        }  
+        }
+
+        protected ExtractorParameters CreateDefaultExtractorParameters(
+               string sourceApimName = MockSourceApimName,
+               string destinationApimName = MockDestinationApimName,
+               string resourceGroup = MockResourceGroup,
+               string fileFolder = MockFileFolder,
+               string apiName = MockApiName,
+               string multipleAPIs = null,
+               string linkedTemplatesBaseUrl = null,
+               string linkedTemplatesSasToken = null,
+               string linkedTemplatesUrlQueryString = null,
+               string policyXmlBaseUrl = null,
+               string policyXmlSasToken = null,
+               string splitAPIs = null,
+               string apiVersionSetName = null,
+               string includeAllRevisions = null,
+               string baseFileName = null,
+               ServiceUrlProperty[] serviceUrlParameters = null,
+               string paramServiceUrl = null,
```

Comment:
Discussion notes: 
1. Method currently will return ExtractorConsoleAppConfiguration instead of parameters
2. GetMockedExtractorConsoleAppConfiguration method will be deprecated in different PR and tests will be using newly introduced method, which provides more flexibility in terms of configurations for test purposes.

Reply:
thanks, it looks great now!